### DISCUSSION TITLE
Stable Release Aliens

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8317] [Q&A] [answered]
Which aliens are in 9.16.1 (or the stable release)?

**Comment by RisingLeaf** [comment, marked as answer]
Alien species in 0.9.16.1:
- Quarg
- Remnant 
- Hai
- Wanderer
- Coalition(Kimek, Arach and Saryd)
- Korath
- Kahet
- Korath Automata
- Pug
- Archons

Extinct Species but mentioned:
- Sheragi
- Builders

Alien species upcoming hopefully for next stable:
- Gegno (already in)
- Bunrodea
- Xapleux 
- Watchers

Other unconfirmed species I know of:
- Avgi
- Searchers
- Incipias 
- A lot more

**Reply by SultanRTG** [reply]
Do you mean that the Gegno is in the pre-release already?

**Reply by Hecter94** [reply]
> Do you mean that the Gegno is in the pre-release already?

Yes they are
